lib/schema.ts

```diff
@@ -840,21 +846,23 @@ const RawConfigSchema = z
       })
       .optional(),
     deployRag: z.boolean().optional().default(false),
+    deployChat: z.boolean().optional().default(false),
+    deployUi: z.boolean().optional().default(false),
```

Comment:
This would potentially be a breaking change for people who have already deployed LISA with default options. For the next release, we can add notes to ensure people specify these if they've previously deployed, just so they have a breaking change warning.

Reply:
I can make them default to `true` which should eliminate the breaking change issue, right?

Reply:
Admittedly, this was likely just a copy/paste oversight from the `deployRag` schema.